Form/form_us.xml

```diff
@@ -2403,7 +2403,7 @@
                    <_attribute>Months unemployed during census year</_attribute>
                </column>
                <column>
-                   <_attribute>(Day of the Enumerator’s visit) sick or temporarily disabled</_attribute>
+                   <_attribute>(Day of the Enumerator's visit) sick or temporarily disabled</_attribute>
```

Comment:
Sorry, @Nick-Hall , I missed your comment. I just had the feeling that a non-ASCII character is not optimal (producing error message) but more importantly, I find it hard to distinguish between " ` ", " ' " and " ´ ". Usually, I stick to " ' " and let the software do the right thing. In this case, I don't care because I will probably never use this entry. Please just say what you think and I update the PR accordingly.